paraatm/simulation_method/integrated_flight_health_monitoring.py

```diff
@@ -0,0 +1,302 @@
+"""
+NASA University Leadership Initiative program
+Information fusion for real-time national air transportation system prognostics under uncertainty
+
+    Integrated Flight Heatlrh Monitoring (IFHM) module
+    - Simulate normal flight operation
+    - Simulate flight with upset flight scenarios using 1) upgen sub module
+    - Monitor flight health in real-time and detect aircraft upsets
+      using 2) fhm sub module
+    - Generate output files to perform analysis
+
+    @author: Hyunseong Lee, Adaptive Intelligent Materials & Systems (AIMS) Center,
+             Arizona State University
+    
+    Last modified on 5/25/2020
+"""
+
+import time
+import os
+from jpype import *
+from array import *
```

Comment:
In general, please avoid using `import *`.  It makes it more difficult to maintain the code because you cannot tell where functions are coming from.

In this particular case, you may be able to remove these altogether.  I'm not sure whether you are really using the `array` module (numpy provides much of the same functionality).  Also, you probably do not need to call jpype directly in your module (I will add more detail about this separately).